src/commands/CreateEnvironmentVariableCommand.ts

```diff
@@ -21,25 +21,27 @@ import * as vscode from 'vscode';
 import { DataVirtConfig, ConfigMapRef, SecretRef, Property, KeyRef } from '../model/DataVirtModel';
 import { EnvironmentTreeNode } from '../model/tree/EnvironmentNode';
 
+const CREATE_NEW_ENTRY: string = '...New...';
```

Comment:
Seems strange to have the first 3 dots. Why not just 'New..'?

Reply:
That was more a protective measure as I wanted to ensure that this item is always on top. I haven't tested yet if the quickpicks are sorted in some way. If that is not the case then I will rename it to just "New...".

Reply:
ok, understand the use case.

I'm surprised that VS Code doesn't allow that. I've seen several places with group separators that would do the trick but... it is not part of API!!! https://github.com/microsoft/vscode/issues/74967
a specific sort seems not possible either: https://github.com/microsoft/vscode/issues/63050

Reply:
I renamed it to "New..." and will revisit it when doing the logic part for filling that quickpick.